CMakeLists.txt

```diff
@@ -210,10 +210,10 @@ if (NGF_BUILD_SAMPLES STREQUAL "yes")
 
     # Determine the backend to build the samples with based on the platform.
     if (WIN32)
-        set(NGF_SAMPLES_BACKEND nicegraf_vk)
+        set(NGF_SAMPLES_BACKEND nicegraf-vk)
         set(NGF_SAMPLES_PLATFORM_SOURCE ${CMAKE_CURRENT_LIST_DIR}/samples/common/platform/windows/window.cpp)
     else()
-        #set(NGF_SAMPLES_BACKEND nicegraf_mtl)
+        #set(NGF_SAMPLES_BACKEND nicegraf-mtl)
```

Comment:
should we also remove this commented line ?

Reply:
yup